admin/includes/languages/lang.english.php

```diff
@@ -534,6 +534,7 @@
     'TEXT_BOOLEAN_VALIDATE' => 'The value is required to be a boolean value or equivalent.',
     'TEXT_ASC' => 'asc',
     'TEXT_DESC' => 'desc',
+    'TEXT_DOWNLOADABLE_PRODUCTS_MISCONFIGURED' => 'Some downloadable products are misconfigured.  See Admin &gt; Catalog &gt; Downloads Manager.',
```

Comment:
Is this really the "right" message? What about if the result was intentional or had no impact on the site's normal operation? Wouldn't it be more assistive to indicate the result of this configuration? Such as:
> Configuration of some downloadable product causes free shipping for all cart items. See Admin &gt; Catalog &gt; Downloads Manager.

I say this because for example there are carts that do not ship anything and configuring the product causing this message will have no direct impact on the income to the site (until product is incorporated requiring shipping fee collection not associated with the free shipping module).

Reply:
Yes, probably worth making this suggested change.